recipes/mantid/meta.yaml

```diff
@@ -57,6 +57,7 @@ requirements:
     - nexus
     - {{ pin_compatible("numpy") }}
     - {{ pin_compatible("occt", max_pin="x.x.x") }}
+    - pycifrw
```

Comment:
Currently `pycifrw` is pinned to `4.4.1` in developer builds as any later versions cause the `LoadCIF` algorithm to error. Should we introduce a similar pin here?